internal/controller/nonadminbackup_controller.go

```diff
@@ -89,6 +89,8 @@ func (r *NonAdminBackupReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, err
 	}
 
+	_, syncBackup := nab.Labels["openshift.io/oadp-nab-synced-from-nacuuid"]
```

Comment:
this was my idea to differ NABs created by user and by sync controller: adding a label to object

what you think about this approach?

Reply:
That sounds like a proper design for that. Would use something from `https://github.com/openshift/oadp-operator/blob/master/api/v1alpha1/oadp_types.go#L35` and we are using it in few places already in the constant.go

Possibly adding yet one label, but that would mean the object is managed by entirely and I think that may be not true, as once it's synced it's no longer managed by sync controller it's the user responsibility to do something with it like deletion:
```
app.kubernetes.io/managed-by: self-service-sync-controller
```